src/plots/barPlot.ts

```diff
@@ -116,8 +124,8 @@ export class Bar<X, Y> extends XYPlot<X, Y> {
    *
    * @return "vertical" | "horizontal"
    */
-  public orientation() {
-    return this._isVertical ? Bar.ORIENTATION_VERTICAL : Bar.ORIENTATION_HORIZONTAL;
+  public orientation(): BarOrientation {
+    return this._isVertical ? "vertical" : "horizontal";
```

Comment:
any way to use constants here instead of all these string literals?

i expected something like https://github.com/palantir/blueprint/issues/324